osler/inventory/tests.py

```diff
@@ -0,0 +1,110 @@
+from django.test import TestCase
+from django.urls import reverse, resolve
+from osler.inventory.models import DrugCategory, MeasuringUnit, Manufacturer, Drug
+from osler.inventory import views, forms
+# Create your tests here.
```

Comment:
What's your code coverage? Make sure all your files are >90%

Reply:
It's 100% for everything!